packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -214,18 +242,30 @@ export default function nodeBuilder(isDiscrete) {
     const layered = state.labels.activeMode === 'layered';
     let majorTickNodes;
 
+    if (settings.arc.isRadial) {
+      buildOpts.startAngle = settings.arc.startAngle;
+      buildOpts.endAngle = settings.arc.endAngle;
+      buildOpts.radius = settings.arc.radius;
+    }
     if (settings.line.show) {
       buildOpts.style = settings.line;
       buildOpts.padding = settings.paddingStart;
-
-      nodes.push(buildLine(buildOpts));
+      if (settings.arc.isRadial) {
+        nodes.push(buildArcLine(buildOpts));
+      } else {
+        nodes.push(buildLine(buildOpts));
+      }
     }
     if (settings.ticks.show) {
       buildOpts.style = settings.ticks;
       buildOpts.tickSize = settings.ticks.tickSize;
       buildOpts.padding = tickSpacing(settings);
-
-      majorTickNodes = tickBuilder(major, buildOpts);
+      if (settings.arc.isRadial) {
```

Comment:
What happens if the settings does not have arc defined. The old versions does not have arc defined in settings.